.github/workflows/minio-image.yml

```diff
@@ -14,21 +14,37 @@ on:
 env:
   ICR_REGISTRY: icr.io
   ICR_NAMESPACE: ai-services-cicd
+  IMAGE_FILE_NAME: image.tar
 
 jobs:
   build-publish-image:
     name: Build and Publish Image
     runs-on: ubuntu-24.04-ppc64le
-
     steps:
       - name: Checkout code
         uses: actions/checkout@v5
-      
+
       - name: Build minio image
         working-directory: images/minio
         run: |
           make build REGISTRY=${{ env.ICR_REGISTRY }}/${{ env.ICR_NAMESPACE }}
 
+      - name: Save minio image to file
+        working-directory: images/minio
+        run: |
+          IMAGE_NAME=$(make image-name REGISTRY=${{ env.ICR_REGISTRY }}/${{ env.ICR_NAMESPACE }} )
+          podman save > ${{ env.IMAGE_FILE_NAME }} $IMAGE_NAME
+      
+      - name: Trivy vulnerability scan
+        id: trivy
+        uses: aquasecurity/trivy-action@0.33.1
+        with:
+          input: images/minio/${{ env.IMAGE_FILE_NAME }}
+          format: table
+          severity: CRITICAL,HIGH,MEDIUM
+          exit-code: 1
+        continue-on-error: true
```

Comment:
@adarshagrawal38 We need to fail the check when any CVE is found. This will always pass.
What we want is try to complete scan to list all CVEs instead of just exiting out at first encounter. If this is not possible it is fine to just fail.